src/main/java/org/opensrp/web/rest/PlanResource.java

```diff
@@ -186,4 +190,22 @@ else if (sName == null && field.getName().equals(fieldName)) {
 		}
 		return false;
 	}
+
+
+	static class PlanSyncRequestWrapper {
+		@JsonProperty
```

Comment:
why not pass the json attribute value on this tag so that the variables are named using camelCase convention